mappings/net/minecraft/block/BubbleColumnBlock.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/unmapped/C_eopzqcts net/minecraft/block/BubbleColumnBlock
 	FIELD f_jarlnoge SCHEDULED_TICK_DELAY I
 	FIELD f_oylcoxnk DRAG Lnet/minecraft/unmapped/C_xhwijdsd;
-	METHOD m_agslpkrg isStillWater (Lnet/minecraft/unmapped/C_txtbiemp;)Z
+	METHOD m_agslpkrg canBubbleColumnExistIn (Lnet/minecraft/unmapped/C_txtbiemp;)Z
```

Comment:
I don't particularly liek this, perhaps `canBeBubbleColumn`?

Reply:
That sounds a lot better :D